portal/server/utils/csv-utils.test.js

```diff
@@ -62,7 +62,7 @@ describe('csv-utils', () => {
       expect(parsedData).toEqual(expectedParsedData);
     });
 
-    it('Parses an excelJS workbook and adds in addresses for any missing cells in a row', async () => {
+    it('Parses an excelJS workbook and adds in null values with addresses for any missing cells in a row', async () => {
```

Comment:
is null technically right now? It's not far off an empty space is basically null in a string :D